### ISSUE TITLE
data normalization

### ISSUE THREAD

**Issue by Z-mingyu** [wholebody3d/wholebody3d#21]
hello, I met some questions in data processing.
how can I convert positions_3d data to normalization data? 
position_2d data can be converted to [-1,1] using cameras' parameters "res_w" and "res_h", but what's the max value of 3d_data?
I'm looking forward to your response. thank you very much .

**Comment by Zhuyue0324** [comment]
If you just need a sample-wise, batch-wise or even dataset-wise normalization of 3d coordinates, just take the max and min value on each axis of that sample, batch or dataset, and you have the scale

**Comment by Z-mingyu** [comment]
when evaluate the model, the output_3d_data value is within [-1,1]. so I should take the max and min value of the 3d_gt_keypoints, then convert output_3d_data to original value. 
Or training the model directly using the original data which don't be converted to [-1,1] is OK?

**Comment by Zhuyue0324** [comment]
In our paper, the benchmark we try with data normalization, we always suppose we don't know the groundtruth scaling of test set. Thus we use the average scaling of 3D on the training set
If you don't want to use it, then no scaling into [-1, 1] is also a choice.

**Comment by Z-mingyu** [comment]
thanks for your quick response. 
the reason I asked former question is that the 3d_data_gt from semGCN and GraphSH had been normalized to [-1,1]. when evaluate the results with mpjpe, they multiplied the Euclidean norm by 1000. so I just think the scale of 3d_data_gt is 1000.
